src/sql/platform/connection/common/connectionManagement.ts

```diff
@@ -71,6 +71,7 @@ export interface IConnectionResult {
 	callStack: string;
 	errorHandled?: boolean;
 	connectionProfile?: IConnectionProfile;
+	connectionUri?: string;
```

Comment:
why is this needed? connectionProfile property should include the connectionUri, right?

Reply:
As I recall, when connecting for the first time, the connectionUri is blank, so if we never connect at all for the first time, we wouldn't have the necessary connectionUri to make the password change. I will have to verify this is the case.

Reply:
The connection profile does not include the URI unfortunately, but we can generate it using the utils